### DISCUSSION TITLE
Cannot found module 'async-mutex'

### DISCUSSION THREAD

**Discussion by mirreri** [microsoft/vscode-discussions#249] [Extension Development QnA] [answered]
I am writing a webview extension with TypeScript, and I add a module 'async-mutex' to package.json
![Dr9L5](https://user-images.githubusercontent.com/22906283/202317686-eca0ab23-6585-462b-a4cc-a8e286c2b273.png)
and everything is fine when I locally test with "Run Extension", however, after I package my project with "vsce package" and install extension with vsix file, I got an error from webview developer tool
![Dr9L5](https://user-images.githubusercontent.com/22906283/202317790-fb4c0ba0-a129-4b66-a9fd-a998eb43f93a.png)
But if I run "npm install" in C:\Users....vscode\extensions<extension-name>, that extension will work fine, but I can't ask everyone to do so, is there any way to fix this problem or maybe I miss some important step in this case?

Appreciate for your help

**Comment by gjsjohnmurray** [comment]
I suggest you list it under the "dependencies" section instead of under "devDependencies".

**Comment by tao-cumplido** [comment, marked as answer]
You need to bundle your extension. See https://code.visualstudio.com/api/working-with-extensions/bundling-extension
